examples/workflows/randomly_failing_workflow.rb

```diff
@@ -0,0 +1,11 @@
+require 'activities/randomly_failing_activity'
+
+class RandomlyFailingWorkflow < Temporal::Workflow
+  def execute
+    RandomlyFailingActivity.execute!
+
+    return 'You are very lucky!'
+  rescue RandomlyFailingActivity::WrongGuess => e
+    return e.message
```

Comment:
You can omit `return`

Reply:
This is a copy/paste job.  The house style seems to be to not emit.